src/features/identityCheck/pages/identification/errors/eduConnect/NotEligibleEduConnect.tsx

```diff
@@ -13,7 +13,57 @@ import { TouchableLink } from 'ui/components/touchableLink/TouchableLink'
 import { PlainArrowPrevious } from 'ui/svg/icons/PlainArrowPrevious'
 import { Typo } from 'ui/theme'
 
-import { useNotEligibleEduConnectErrorData } from '../hooks/useNotEligibleEduConnectErrorData'
+import {
+  NotEligibleEduConnectErrorData,
+  useNotEligibleEduConnectErrorData,
+} from '../hooks/useNotEligibleEduConnectErrorData'
+
+const PrimaryButtonToDisplay = ({
+  button,
+}: {
+  button: NotEligibleEduConnectErrorData['primaryButton'] | undefined
+}): React.ReactNode => {
+  if (!button) {
+    return (
+      <TouchableLink
+        key={1}
+        as={ButtonPrimaryWhite}
+        wording="Retourner à l'accueil"
+        navigateTo={navigateToHomeConfig}
+      />
+    )
+  }
+
+  const { text: primaryButtonText, icon: primaryButtonIcon, onPress: onPrimaryButtonPress } = button
+
+  if ('navigateTo' in button && button.navigateTo) {
```

Comment:
Je comprends pas trop cette condition, c'est quoi la différence entre `if ('navigateTo' in button)` et `if (button.navigateTo)`

Reply:
Vu ensemble : le `('navigateTo' in button)` permet de vérifier que la clef est présente dans l'objet et le `(button.navigateTo)` que sa valeur n'est pas undefined